app/controllers/api/v1/pages_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::PagesController < Api::V1::ApplicationController
+  skip_before_action :authenticate, only: [:home]
+
+  def home
+    @products = Product.all
+    @products = @products.where(category: params[:category]) if params[:category].present?
+    @products = @products.where(vegetarian: params[:vegetarian]) if params[:vegetarian].present?
+    @products = @products.order(price: params[:order]) if params[:order].present?
+
+    render json: @products
+  end
+
+  def help; end
```

Comment:
do you need these methods?

Reply:
i have some static pages. for their routes these methods are needed